django/template/loader_tags.py

```diff
@@ -207,6 +210,11 @@ def render(self, context):
         except Exception:
             if context.template.engine.debug:
                 raise
+            else:
+                template_name = getattr(context, 'template_name', 'unknown')
+                logger.warn('Exception raised while rendering {{% include %}} for '
```

Comment:
Instead of interpolating the parameter in the message, logging should do that work, so this should be:
```python
logger.warn(
    'Exception raised while rendering {%% include %%} for '
    'template %s. Empty string rendered instead.',
    template_name,
    exc_info=True,
)
```
I am not sure if template name retrieval works in the 'unknown' case (seems to return "None" in the test case). Would be nice to have a test for the case when a template name does exist too.
Edits: http://dpaste.com/1XTAAT0